lib/extendOrdWithCustom.js

```diff
@@ -0,0 +1,86 @@
+const path = require("path");
+const cds = require("@sap/cds");
+const _ = require('lodash');
+
+const ORD_ENV_KEY = 'ord';
+const CONTENT_MERGE_KEY = 'ordId'
+
+function hasStringConflictsWithCdsrcJson(key) {
+    if (key in cds.env[ORD_ENV_KEY]) {
+        console.error(`Detect ord custom content has conflict with .cdsrc.json: ${key},
+            please resolve it, use .cdsrc.json value.`);
+        return true;
+    }
+    return false;
+}
+
+function handleStringConflicts(key) {
+    if (hasStringConflictsWithCdsrcJson(key)) {
+        return;
+    }
+    console.error(`Detect ord custom content has conflict with default value: ${key},
+        please define it in .cdsrc.json.`);
```

Comment:
Why would we need to run into this condition? Isn't the point of custom ORD content / overrides to be able to override default values?

Reply:
dead code?